tests/integration/test_docker.py

```diff
@@ -338,3 +349,13 @@ def test_run_with_volumes_metadata_check():
         assert mount["Mode"] == "Z"
 
         container.delete(force=True)
+
+
+def test_list_containers():
```

Comment:
nit: there could be another check - add new container and check len is incremented

Reply:
on it

Reply:
updated